docs/topics/http/urls.txt

```diff
@@ -106,7 +106,8 @@ Example requests:
 * ``/articles/2003/`` would match the first pattern in the list, not the
   second one, because the patterns are tested in order, and the first one
   is the first test to pass. Feel free to exploit the ordering to insert
-  special cases like this.
+  special cases like this. Here, Django simply calls the function
+  views.special_case_2003(request) only passing along the request.
```

Comment:
add double backticks around views.special_case_2003(request) so it renders as a code block